StyleSheets/StyleSheets/SearchBarStyleSheet.swift

```diff
@@ -0,0 +1,66 @@
+import UIKit
+
+open class SearchBarStyleSheet: ViewStyleSheet<UISearchBar> {
+    public var height: CGFloat
+    public var showsCancelButton: Bool
+    public var searchTextPositionAdjustment: UIOffset
+    public var keyboardType: UIKeyboardType
+    public var returnKeyType: UIReturnKeyType
+    public var enablesReturnKeyAutomatically: Bool
+
+    public init(
+        backgroundColor: UIColor = .white,
+        cornerRadius: CGFloat = 10,
+        height: CGFloat = 36,
+        showsCancelButton: Bool = false,
+        searchTextPositionAdjustment: UIOffset = UIOffset(horizontal: 8, vertical: 0),
+        keyboardType: UIKeyboardType = .default,
+        returnKeyType: UIReturnKeyType = .search,
+        enablesReturnKeyAutomatically: Bool = true
+        ) {
+        self.height = height
+        self.showsCancelButton = showsCancelButton
+        self.searchTextPositionAdjustment = searchTextPositionAdjustment
+        self.keyboardType = keyboardType
+        self.returnKeyType = returnKeyType
+        self.enablesReturnKeyAutomatically = enablesReturnKeyAutomatically
+        super.init(backgroundColor: backgroundColor, cornerRadius: cornerRadius)
+    }
+
+    open override func apply(to element: UISearchBar) {
+        super.apply(to: element)
+        element.setTextInputBackgroundColor(color: backgroundColor ?? .clear,
+                                            height: height,
+                                            cornerRadius: cornerRadius)
```

Comment:
Why do we need this

Reply:
Because on `UISearchBar` one can only set the background image, not a colour.